test/flows/vat_payment_deadlines_flow_test.rb

```diff
@@ -0,0 +1,85 @@
+require "test_helper"
+require "support/flow_test_helper"
+
+class VatPaymentDeadlinesFlowTest < ActiveSupport::TestCase
+  include FlowTestHelper
+
+  setup do
+    WebMock.stub_request(:get, WorkingDays::BANK_HOLIDAYS_URL)
+      .to_return(body: File.open(fixture_file("bank_holidays.json")))
+    testing_flow VatPaymentDeadlinesFlow
+  end
+
+  should "render a start page" do
+    assert_rendered_start_page
+  end
+
+  context "question: when_does_your_vat_accounting_period_end?" do
+    setup { testing_node :when_does_your_vat_accounting_period_end? }
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of how_do_you_want_to_pay? for a date of '2013-04-30' response" do
+        assert_next_node :how_do_you_want_to_pay?, for_response: "2013-04-30"
+      end
+
+      context "validation" do
```

Comment:
this validation block is conventionally be outside of the `next_node` context and before it. As per: https://github.com/alphagov/smart-answers/blob/8d4d04d61c3ede76358fb10cf2470e94d4fa68a1/test/flows/state_pension_age_flow_test.rb#L35-L47